packages/devtools_app/lib/src/vm_service_wrapper.dart

```diff
@@ -407,6 +407,18 @@ class VmServiceWrapper implements VmService {
     }
   }
 
+  Future<HttpTimelineLoggingState> getHttpEnableTimelineLogging(
```

Comment:
both of these methods should check for `isProtocolVersionSupported` so that we are backwards compatible with older vm_service versions

Reply:
What would we want to do in those situations? Raise a dialog telling users to ensure `HttpClient.enableTimelineLogging` is set to true in their code?

Reply:
I think we wouldn't show the networking page in devtools if the user is on a lower version of the vm_service than required. Aren't these RPCs crucial to the functionality of the networking page?

Reply:
Yes and no. These RPCs are just used to get and set the state of network profiling, but users on Flutter 1.12 and Dart 2.7 can already enable HTTP request logging to the timeline by setting `HttpClient.enableTimelineLogging` manually. The RPCs allow for `HttpClient.enableTimelineLogging` to be inspected / set over the service protocol.